src/object/schedcontext.c

```diff
@@ -378,6 +375,25 @@ void schedContext_unbindNtfn(sched_context_t *sc)
     }
 }
 
+void schedContextMaybeUnbindNtfn(notification_t *ntfnPtr)
+{
+    sched_context_t *boundSchedContext;
+    boundSchedContext = (sched_context_t *)notification_ptr_get_ntfnSchedContext(ntfnPtr);
```

Comment:
Any reason why `SC_PTR` isn't used here?

Reply:
We should prefer the macro (`SC_PTR`) when we can.